exam-reg-management.yaml

```diff
@@ -0,0 +1,221 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Examination Regulation API for UC4."
+  version: "0.12.1"
+  title: "UC4"
+
+servers:
+  - url: https://uc4.cs.upb.de/api/production/examreg-management
+
+tags:
+- name: "Version Management"
+  description: "Everything about the Version"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+- name: "Examination Regulation Management"
+  description: "Everything about the configuration"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+
+paths:
+  /version:
+    get:
+      tags:
+      - "Version Management"
+      summary: "Get version"
+      description: "Returns the version of the current service"
+      operationId: "getVersion"
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JsonVersionNumber'
+  /examination-regulations:
+    get:
+      tags:
+      - "Examination Regulation Management"
+      summary: "Get examination regulations"
+      description: "Get all examination regulations, or the ones specified by the query parameter"
+      operationId: "getExaminationRegulation"
+      parameters:
+      - name: "regulations"
+        in: "query"
+        description: "A comma seperated list of examination regulations to filter by"
+        required: false
+        schema:
+          type: "string"
+      - name: "active"
+        in: "query"
+        description: "A flag determining, if only active or inactive examination regulations should be fetched"
+        required: false
+        schema:
+          type: "string"
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                type: "array"
+                items:
+                  $ref: '#/components/schemas/ExaminationRegulation'
+              examples:
+                ConfigurationExample:
+                  $ref: '#/components/examples/ExampleExaminationRegulations'
+  /examination-regulations/modules:
+    get:
+      tags:
+      - "Examination Regulation Management"
+      summary: "Get all examination regulations"
+      description: "Get all examination regulations which were previously used, or are currently used in the system"
```

Comment:
Sounds like we are getting examination-regulations instead of modules.

Reply:
Good find! This is now changed to a fitting text.